python/cudf/cudf/_lib/pylibcudf/strings/find.pyx

```diff
@@ -0,0 +1,255 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+from libcpp.memory cimport unique_ptr
+from libcpp.utility cimport move
+
+cimport cudf._lib.cpp.strings.find as cpp_find
```

Comment:
This doesn't really matter, but I think in most of pylibcudf we instead have
```suggestion
from cudf._lib.cpp.strings cimport find as cpp_find
```
Either should be fine but it would be nice to stay consistent.